google-cloud-vision/samples/v1/samplegen_basics.rb

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "samplegen_basics")
+
+# sample-metadata
+#   title: This is the sample title
+#   description: This is the sample description
+#   bundle exec ruby samples/v1/samplegen_basics.rb [--display_name "This is the default value of the display_name request field"]
+
+# This is the sample description
+def sample_create_product_set display_name = "This is the default value of the display_name request field"
+  # [START samplegen_basics]
+  # Import client library
+  require "google/cloud/vision"
+
+  # Instantiate a client
+  product_search_client = Google::Cloud::Vision::ProductSearch.new version: :v1
```

Comment:
ℹ️ `FYI` Recommend `replace()`-ing explicit versions with implicit (_for samples which use the default version_)

When adding samplegen to Vision, for example, we would want to update `synth.py` with:

```
s.replace("samples/v1/*.rb", r".new version: :v1", ".new")
```